go/libkb/util_windows.go

```diff
@@ -184,3 +188,47 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	}
 	return nil
 }
+
+// Notify the shell that the thing located at path has changed
+func notifyShell(path string) {
+	shChangeNotifyProc.Call(
+		uintptr(0x00002000), // SHCNE_UPDATEITEM
+		uintptr(0x0005),     // SHCNF_PATHW
+		uintptr(unsafe.Pointer(syscall.StringToUTF16Ptr(path))),
+		0)
```

Comment:
Oh man, is there some library out there that wraps these sorts of calls for us? >.<